src/permissions.js

```diff
@@ -50,15 +50,13 @@ const getOrganizationalUserOrganizationPermissions = logger => {
 }
 
 /**
- * @description Validate permissions
- * This function receives an array of validators, and tries to validate some of them, i.e.,
- * each validator can be an array or a regular expression. When it's an array, the
- * validator requires the user to have all of its permissions. When it's a regular
- * expression, the validator requires some user permission to match it.
- * @param {Array.<Array.<String>|RegExp>} validators Validators
+ * Permission middleware factory
+ * @param {*} logger
+ * @returns A middleware that checks if the user in req.user has the permissions
+ * to continue the request to the next handler.
  */
-function validateSomePermissionCluster (logger) {
-  return (validators = []) => {
+function validateSomePermissionClusterMiddleware (logger) {
+  return validators => {
```

Comment:
Por que removeu a atribuição default de array vazio?

Reply:
Bom ponto. Tá mais pra "não adicionar" do que "remover" (mas explicar não vale a pena)
```suggestion
  return (validators = []) => {
```